docker/entrypoint.sh

```diff
@@ -9,14 +9,14 @@ if [ "$ENV" != 'dev' ]; then
 fi
 
 # Run database migrations in RQ container
-if [ "$IS_RQ" = 'yes' ]; then
+if [ "$SERVICE_NAME" = 'rq' ]; then
   echo ">> Running database migrations."
   python manage.py migrate --no-input
   echo ">> Verifying settings."
   python manage.py verify_settings
   echo ">> Starting RQ worker(s)"
 
-else
+elif [ "$SERVICE_NAME" = 'app' ]; then
```

Comment:
We may as well add a custom message for when the scheduler is starting up

```suggestion
elif [ "$SERVICE_NAME" = 'rq-scheduler' ]; then
  echo ">> Starting RQ scheduler"

elif [ "$SERVICE_NAME" = 'app' ]; then
```

Reply:
That's a good idea.